README.md

```diff
@@ -1,7 +1,49 @@
 # hasura-bot :robot:
 
-Hasura's own GitHub bot.
+Hasura's own GitHub bot, currently tasked with commenting on pull requests when they are opened/closed/merged.  
+Deployed on [Zeit Now](https://zeit.co/now): https://hasura-github-bot.now.sh
 
-Currently tasked with commenting on pull requests when they are opened/closed/merged.
+## Local Setup
+First, you will need to create an account on [vercel](https://vercel.com/signup).
 
-Deployed on [Zeit Now](https://zeit.co/now): https://hasura-github-bot.now.sh
+Next, set up `ngrok` by following the instructions given [here](https://ngrok.com/download).
+
+Now that you have both of these things set up,
+- Fork this repo and clone it on your system:
+  ```bash
+  $ git clone https://github.com/<username>/github-bot
+  ```
+- Navigate into the repo and install the dependencies:
+  ```bash
+  $ cd hasura-bot
+  $ npm install
+  ```
+- Create a `.env` file and set `GITHUB_TOKEN` and `WEBHOOK_SECRET` env variables.
+- Start a local server using `vercel`:
+  ```bash
+  $ vercel dev
```

Comment:
Let's use `npm run dev` instead of directly using vercel. This will help us ensure the version of vercel command by taking it from the dev dependencies of package.json.